Uplift/Views/Home/FitnessCenterView.swift

```diff
@@ -126,6 +145,79 @@ struct FitnessCenterView: View {
         }
         .frame(minWidth: 84, alignment: .leading)
     }
+    private var equipmentSection: some View {
+
+        VStack(spacing: 12) {
+            sectionHeader(text: "EQUIPMENT")
+            ScrollView(.horizontal) {
+                HStack(spacing: 12) {
+                    ForEach(EquipmentType.allEquipmentTypes, id: \.self) { equipmentType in
+                        if eqpmtTypeDict[equipmentType] != 0 {
+                            VStack(alignment: .leading) {
+                                Text(equipmentTypeToString(eqmtType: equipmentType))
+                                    .font(Constants.Fonts.h3)
+                                    .padding()
+                                    .fixedSize(horizontal: true, vertical: false)
```

Comment:
Are we going to cut off the text or wrap it on a new line if it overflows? We should speak with design about this.

Reply:
Reminder to self to check back in with design; they want some slack on each line and space between the end of the description and quantity

Reply:
Ok I spoke to design right now and they said to wrap it onto a new line if it overflows, but my current wrapping doesn't leave enough space between the description of the equipment and the quantity